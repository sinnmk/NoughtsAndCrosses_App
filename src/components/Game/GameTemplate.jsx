import React from 'react';
import {Button} from 'reactstrap';
import Home from '../Home/Home';
import {Link} from 'react-router-dom'

const GameTemplate = () =>{
    return (
        <div>
            <Link to="/" onClick={Home}><Button className = 'back-button'>Back to Main Menu</Button></Link>
        </div>
    )
}
export default GameTemplate;