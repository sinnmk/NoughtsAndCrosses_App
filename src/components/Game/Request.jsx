import React, {Component} from 'react';

class GameRequest extends Component{


}

export default GameRequest;