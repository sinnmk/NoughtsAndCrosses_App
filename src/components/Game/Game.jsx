import { Component } from 'react';
import GameTemplate from './GameTemplate'; 

export default class Game extends Component {
    constructor (props) {
        super(props);
        this.onRadioBtnClick = this.onRadioBtnClick.bind(this);
      }
    
      onRadioBtnClick(rSelected) {
        this.setState({ rSelected });
      }
    
      render() {
        return GameTemplate.call(this);
      }
    }