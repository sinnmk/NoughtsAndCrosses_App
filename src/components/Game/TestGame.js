import React from 'react';
import {shallow, mount, render} from 'enzyme';
import Game from ".Game";

describe("Game", () => {
    it("can render the Game component", () => {

    });
});