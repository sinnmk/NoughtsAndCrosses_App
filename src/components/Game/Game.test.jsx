import React from 'react';
import renderer from 'react-test-renderer';
import Game from './Game';

describe('When Game is loaded', () =>{
    test('Game content is rendered', () =>{
        const component = renderer.create(
            <Game/>).toJSON();

        expect(component).toMatchSnapshot();
    })
})