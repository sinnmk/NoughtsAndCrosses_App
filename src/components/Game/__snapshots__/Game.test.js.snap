// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When Game is loaded Game content is rendered 1`] = `
<div>
  <div>
    Board
  </div>
</div>
`;
